docs/building-your-quest/actions/network-http-request.md

```diff
@@ -40,20 +40,20 @@ These can be accessed using:
 ```yaml
 do:
 - actionId: network_http_request
-  name: call_heroku_backend
+  name: call_backend
   params:
-    url: "https://${user.backendHerokuAppName}.herokuapp.com/api/items?limit=10"
+    url: "${user.K8sBackendUrl}/api/items?limit=10"
 if:
   conditions:
   - conditionId: is_truthy
     params:
-      value: ${outputs.call_heroku_backend.data?.items}
+      value: ${outputs.call_backend.data?.items}
 
   then:
     ...
 ```
 
-In this example we make a request to users Heroku server to get items. Then we check that response data includes `items`, which means the request was successful.
+In this example we make a request to users K8s server to get items. Then we check that response data includes `items`, which means the request was successful.
```

Comment:
@NivRichter Can we even say a "K8s server"? Shouldn't it be a cluster or a service?

```suggestion
In this example we make a request to a user's K8s server to get items. Then we verify that response data includes `items`, which means the request was successful.
```

Reply:
That's an interesting point, I think we better to say `In this example we make a request to the user's K8s hosted backend to get items`